content/company/strategy/code-graph/batch-changes/index.md

```diff
@@ -46,6 +46,12 @@ To deliver our vision, we will:
 - Then, release Batch Changes on Sourcegraph Cloud, and test and learn how we can serve individuals and smaller teams.
 - Eventually, address the problem end to end by becoming the go-to place for code change tools and recipes.
 
+## Competitive landscape
```

Comment:
This is really good, thanks for adding @malomarrec ! Thanks for summarizing and including in your strategy.